src/EncompassRest/Loans/LoanOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using EncompassRest.Utilities;
+
+namespace EncompassRest.Loans
+{
+    public abstract class LoanOptions
+    {
+        public bool Populate { get; set; }
```

Comment:
we should have defaults for this. - false

Reply:
If you don't assign a value to it the default is `false`.

Reply:
ok

Reply:
I could make it explicit though I suppose.